src/common/models/Projections.ts

```diff
@@ -124,6 +124,11 @@ export class Projections {
       reverseList.some(level => level === Level.LOW)
     ) {
       return Level.HIGH;
+    } else if (
```

Comment:
Hrm...  I had suggested a change to this code [here](https://github.com/covid-projections/covid-projections/commit/5729b6f44b3c7771bd5a4680974bb6874c82534d#r39517842) bit it may have gotten hidden due to force-pushing...

Basically I think we need to swap MEDIUM_HIGH and MEIDUM when using the reverseList.  As-is, for example, Kansas is getting ranked yellow when it should be orange (because contact tracing is "low" which is orange):

![image](https://user-images.githubusercontent.com/206364/83445264-87e3d780-a401-11ea-9314-31360173d9a8.png)

And on the flip side, Wyoming is orange even though it should be yellow based on the metrics:
![image](https://user-images.githubusercontent.com/206364/83446597-c8445500-a403-11ea-8fd6-1ac637548822.png)

FWIW- I think this is all super confusing and should probably be reworked somehow.

Reply:
Updated! I probably lost the change with a bad rebase. I added it back and confirmed that it works well for the states above.